### PR TITLE
Add tweet status

### LINKED ISSUE #15: Add tweet status
Add a tweet status of the form - 

Case 1: comparing models
One discharge:
Comparing {model1.name}, {model2.name}, and {model3.name} with {parameters} for a {x}C discharge at {y} degrees C
[Link to code]
Experiments:
Comparing {model1.name}, {model2.name}, and {model3.name} with {parameters} for the following experiment: [experiment]

Case 2: varying a parameter
{model.name} with {parameters.name} varying {parameter.name} for a {x}C discharge at {y} degrees C
[Link to code]
{model.name} with {parameters.name} varying {parameter.name} for the following experiment: [experiment]

The "Link to code" option will be added once #14 is closed.

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Closes #15 

Added all the possible tweet statuses, with tests, for all the plots present in the bot right now.

### CODE REVIEW

File: bot/information/information.py [hunk 1]
```diff
@@ -1,47 +1,82 @@
...
-            + str(number)
-            + " "
-            + str(model.name)
-            + " "
-            + str(chemistry["citation"])
-        )
-        return tweet_text
+    params = pybamm.ParameterValues(chemistry=chemistry)
+    c_rate = (
+        params["Nominal cell capacity [A.h]"] / params["Current function [A]"]
```
Comment (rtimms) [inline bot/information/information.py:29]: should this fraction be the other way? higher current should mean a higher c-rate

Comment (brosaplanella) [reply bot/information/information.py:29]: Yup, you are right. Sorry Saransh, got it the wrong way around

Comment (Saransh-cpp) [reply bot/information/information.py:29]: No worries, I'll update this

**Comment by Saransh-cpp** [comment]
@tinosulzer @brosaplanella @rtimms should I go ahead and merge this branch?

**Review by brosaplanella — APPROVED** [review]
Looks good, thanks Saransh!

**Pull request merged**